services/learn-card-network/brain-service/src/routes/profiles.ts

```diff
@@ -684,29 +696,66 @@ export const profilesRouter = t.router({
                     'This route creates a one-time challenge that an unknown profile can use to connect with this account',
             },
         })
-        .input(
-            z
-                .object({
-                    challenge: z.string().optional(),
-                    expiration: z.number().default(3600 * 24 * 30),
-                })
-                .optional()
-        )
-        .output(z.object({ profileId: z.string(), challenge: z.string() }))
+        .input(z.object({
+            expiration: z.enum(['1s', '1h', '24h', '7d', '30d', 'never']).optional().default('1h'),
```

Comment:
This should go back to a number! It prevents a breaking API change and (IMO) it _should_ be a number! We should only be arbitrarily limiting the length in LCA, _not_ in the LCN API!